aurora.dme

```diff
@@ -6,6 +6,8 @@
 // BEGIN_FILE_DIR
 #define FILE_DIR .
 #define FILE_DIR "code"
+#define FILE_DIR "code/modules"
```

Comment:
What is this, and why is this a thing? Please clean up the .dme before we pull this.